.github/workflows/linters.yml

```diff
@@ -0,0 +1,44 @@
+name: Linters
+
```

Comment:
- [ ] The frontend styling is vastly different from the required design:

![image](https://github.com/Swarzstein/final-capstone/assets/57421823/9b326572-4e8e-4e53-bf42-55bf5afa0540)

![image](https://github.com/Swarzstein/final-capstone/assets/57421823/043240b3-5000-4869-8b37-15b3a2c31783)


Kindly, add the required styling to make it look like in the required design. 👍 

- [ ] As you commented in your description, there is no mobile version yet.

- [ ] As you commented in your description, there is no adding items functionality yet.

- [ ] Remember:  Reservations and deletions are not a requirement for a group of 2 people.  However, if you implement them, they need to work well. 👍